### PR TITLE
Add vse.cz (Vysoká škola ekonomická v Praze)

### PR DESCRIPTION
**Pull request by lpavlicek**

### CODE REVIEW

File: entries/v/vse.cz.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "tfa": [
+      "totp",
+      "sms",
+      "u2f",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Microsoft Authenticator"
+    ],    
+    "documentation": "https://ci.vse.cz/english/services/insis/multi-factor-authentication/",
```
Comment (Carlgo11) [inline entries/v/vse.cz.json]: The documentation page only mentions TOTP authentication

File: img/v/vse.cz.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="170.467" width="267.293" xml:space="preserve"><path style="fill:#00adef;fill-opacity:1;fill-rule:nonzero;stroke:none" d="M1200.39 1131.03h795.85V877.77h-398.92V717.168h378.08V464.656h-378.08V284.758h407.38V23.648h-804.31V1131.03M660.535 844.711c0 94.769 59.07 171.129 145.418 199.109L685.746 745.863c-15.933 26.481-25.211 58.578-25.211 98.848zm84.367-159.539 148.95 370.038c65.156-2.49 119.608-23.43 219.388-84.03v94.04c-80.87 47-160.127 69.71-240.888 69.71-164.579 0-298.676-129.71-298.676-293.469 0-88.731 29.351-147.676 73.711-190.68l-183.496-454.82-380.54 935.499H0L451.324 14.379h23.578l236.946 588.656c146.558-85.199 357.172-97.98 357.172-303.094 0-124.839-96.106-220.492-219.524-220.492-80.769 0-146.305 29.18-262.125 118.34V97.262C678.805 32.422 768.727 0 854.063 0c169.157 0 301.847 132.949 301.847 303.184 0 285.886-269.594 284.105-411.008 381.988M1064.25 1200.27H680.051v78.27h384.199v-78.27" transform="matrix(.13333 0 0 -.13333 0 170.467)"/></svg>
```
Comment (Carlgo11) [inline img/v/vse.cz.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 267.29 170.47"><path fill="#00adef" d="M160.05 19.68h106.1v33.76h-53.18v21.41h50.41v33.67h-50.41v23.98h54.32v34.82H160.05V19.68M88.07 57.85c0-12.64 7.88-22.82 19.39-26.55L91.43 71.03c-2.12-3.53-3.36-7.81-3.36-13.18zm11.25 21.27 19.86-49.34c8.68.33 15.95 3.12 29.24 11.2V28.45c-10.77-6.27-21.34-9.3-32.11-9.3-21.94 0-39.82 17.3-39.82 39.13 0 11.83 3.91 19.69 9.83 25.42l-24.47 60.64L11.11 19.61H0l60.17 148.94h3.15l31.6-78.48c19.53 11.36 47.61 13.06 47.61 40.41 0 16.64-12.81 29.4-29.27 29.4-10.77 0-19.5-3.89-34.95-15.78v13.4c12.2 8.65 24.18 12.97 35.56 12.97 22.55 0 40.25-17.73 40.25-40.42 0-38.12-35.95-37.88-54.8-50.93m42.57-68.69H90.67V.02h51.22v10.43"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/v/vse.cz.json (hunk 1 above)
Comment (lpavlicek) [reply entries/v/vse.cz.json]: I have replaced the legacy documentation page with a new one describing all supported 2FA methods.

**Review by Carlgo11 — APPROVED** [review]

File: entries/v/vse.cz.json (hunk 1 above)
Comment (Carlgo11) [reply entries/v/vse.cz.json]: Thanks!

**Pull request merged**